CONTRIBUTING.rst

```diff
@@ -257,14 +257,27 @@ Changelog entries in ``doc/changes/latest.rst`` should adhere to the following c
 
 - Entry in the appropriate category
 - Single line per entry
+- Add a "badge" corresping to the change type (see below)
 - Finish with a link to the PR and the author's profile
-- New contributors to add their details to the ``authors`` section of the ``CITATION.cff``
+- New contributors to add their details to the ``authors`` section of the ``CITATION.cff`` (see below)
+
+List of badges:
+
+.. codeblock:: rst
+
+      :bdg-primary:`Doc`
+      :bdg-secondary:`Maint`
+      :bdg-success:`API`
+      :bdg-info:`Plotting`
+      :bdg-warning:`Test`
+      :bdg-danger:`Deprecation`
+      :bdg-dark:`Code`
```

Comment:
Even if contributors forget them at least we will have them listed somewhere

Reply:
I originally wanted to put this in https://nilearn.github.io/dev/maintenance.html. The idea was that this would be the responsibility of the maintainer during a release so as to not add an extra "burden" to contributors. But maybe it's ok since people seem to already include it sometimes. WDYT? If we keep it here I still don't see a problem with merging a PR without it and then just checking before a release. This should be easy enough since we have to manipulate the file anyways.

Reply:
Ok let's keep it here then

Reply:
Definitely something we can revisit and change if we see that this approach does not work.